src/Workspaces/CoreTest/CodeStyle/EditorConfigCodeStyleParserTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿using Microsoft.CodeAnalysis.CodeStyle;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CodeStyle;
```

Comment:
system should come first.

Reply:
I only touched this file to add the missing banner .. :/

Reply:
you don't have to change. i was just pointing out the problem i saw :)  but, my personal preference is: if you're going to clean up, just clean up :)  I'm in hte habit of just hitting ctrl-r-g when i do this sort of thing.

Reply:
fixed in 1afbe495921f65daf8a1cfea43f5416a6a8ff4fa